server/rpc.go

```diff
@@ -302,6 +309,34 @@ func verifyOrchestratorReq(orch Orchestrator, addr ethcommon.Address, sig []byte
 	return orch.CheckCapacity("")
 }
 
+// authenticateBroadcaster returns an error if authentication fails
+func authenticateBroadcaster(id string) error {
+	if AuthWebhookURL == "" {
```

Comment:
`AuthWebhookURL` is set only for Broadcaster node

Reply:
fixed in 9ce9e44f